infrastructure/docker/build/docker-compose-opt.yml

```diff
@@ -26,3 +26,12 @@ services:
         RHEL_VERSION: ${RHEL_VERSION:-8}
     volumes:
       - ../../../traffic_server/tsb:/opt/tsb-ats:z
+  fakeorigin_build:
+    image: apache/fakeorigin_builder:master
```

Comment:
Can this be `apache/traffic_fakeorigin_builder:master` so that all of the ATC images start with `apache/traffic*_builder`?

Reply:
That's not its name and would be confusing.

Reply:
Also, that convention is already broken with grove.

Reply:
> Also, that convention is already broken with grove.

That convention is not already broken with grove.

https://github.com/apache/trafficcontrol/blob/41669a17927a1d11e8e7f5cbbf930bec2ac216d1/infrastructure/docker/build/docker-compose.yml#L100
https://github.com/apache/trafficcontrol/blob/41669a17927a1d11e8e7f5cbbf930bec2ac216d1/infrastructure/docker/build/docker-compose.yml#L110

Reply:
If a consistent naming convention for all ATC images is desired, it should be `apache/atc_.*_builder` so we also don't conflict with ATS accidentally.  Regardless, I think making that kind of a sweeping change is outside the scope of a single addition.

Reply:
Ah, you're right.  I was looking in the wrong place.  I've updated it since everything else is consistent, but stand by that we should have used a different prefix for this.